internal/ambient/daprsidecar.go

```diff
@@ -0,0 +1,35 @@
+package ambient
+
+import (
+	"context"
+
+	"github.com/dapr/dapr/pkg/injector/sidecar"
+	"k8s.io/client-go/kubernetes"
+)
+
+// DaprSidecarClient represents a client to get Dapr trust bundle.
+type DaprSidecarClient interface {
+	Get(ctx context.Context, ns string) DaprTrustBundle
+}
+
+type daprSidecarClient struct {
+	Clientset *kubernetes.Clientset
+}
+
+// NewDaprSidecarClient creates a DaprSidecarClient instance.
+func NewDaprSidecarClient(cs *kubernetes.Clientset) DaprSidecarClient {
+	return &daprSidecarClient{
+		Clientset: cs,
+	}
+}
+
+// Get gets DaprTrustBundle struct.
+func (s *daprSidecarClient) Get(ctx context.Context, ns string) DaprTrustBundle {
```

Comment:
@mcruzdev we can have just a single init container right? instead of having two .. the same initContainer should load the trust bundles and create the config map.. Does that make sense?

Reply:
With this code, we do not have two initContainers, only one.
We have a Go command that uses the code inside `internal/*` package.